rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -545,7 +545,7 @@ void emu_settings::EnhanceDoubleSpinBox(QDoubleSpinBox* spinbox, SettingsType ty
 	spinbox->setRange(min, max);
 	spinbox->setValue(val);
 
-	connect(spinbox, QOverload<const QString &>::of(&QDoubleSpinBox::valueChanged), [=](const QString&/* value*/)
+	connect(spinbox, QOverload<const QString &>::of(&QDoubleSpinBox::textChanged), [=](const QString&/* value*/)
```

Comment:
might as well replace "value" with "text"

Reply:
Actually changed to the non-deprecated variant of `valueChanged`, the one using a `double`.